recipes/libmysqlclient/all/conanfile.py

```diff
@@ -118,6 +118,8 @@ def _stdcpp_library(self):
 
     def package_info(self):
         self.cpp_info.libs = ["libmysql" if self.settings.os == "Windows" and self.options.shared else "mysqlclient"]
+        self.cpp_info.names["cmake_find_package"] = "MySQL"
```

Comment:
MySQL project doesn't export any target, and I didn't find any official name from the upstream. Do you have a reference from the upstream? Otherwise, I vote for keeping the old name.

Reply:
@uilianries FindMySQL.cmake seems far more frequent than FindMySQLClient.cmake after a fast check on Github. This change has been proposed by @jgsogo

Reply:
@syoliver Indeed mysqlclient is not a good name. But I mean, usually we use lower case if there is no reference about the target, as you should know, letter case matters.

Reply:
@uilianries Yes but Findmysql.cmake with lowercase doesn't exist in github AFAIK. Anyway, I did this PR because I though it was ok to rename. If not, I close this PR, not problem :confused:

Reply:
This pr comes from https://github.com/conan-io/conan-center-index/pull/1626/files#r426721313
The `libmysqlclient` recipe is actually `MySQL` but without building the daemon/client.
So I think `MySQL` makes sense..

Reply:
Where is the daemon/client? Should we build them here too? Maybe we should add an option for CMake's `WITHOUT_SERVER`.

I think `FindMySQL.cmake` is more common, and I personally prefer it over `FindMySQLClient` if I were looking for a library.

Reply:
I believe they are in the source folder, I see no problem adding, but I prefer add it by demand.

So, let's follow FindMySQLClient